mule-extensions-api/src/main/java/org/mule/runtime/extension/api/model/source/ImmutableSourceModel.java

```diff
@@ -56,4 +60,9 @@ public ImmutableSourceModel(String name,
   public boolean hasResponse() {
     return hasResponse;
   }
+
+  @Override
```

Comment:
don't forget that connection providers might want to throw more specific connectivity childs